Adyen/Model/PosTerminalManagement/GetTerminalDetailsResponse.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace Adyen.Model.PosTerminalManagement
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    [DataContract]
+    public class GetTerminalDetailsResponse : Dictionary<string, object>
```

Comment:
Response should be based response parameters and not key value pair. Please check the [documentation] (https://docs.adyen.com/api-explorer/#/postfmapi/v1/post/getTerminalDetails__section_resParams)